src/components/api-request.js

```diff
@@ -1211,8 +1211,15 @@ export default class ApiRequest extends LitElement {
       const contentType = tryResp.headers.get('content-type');
       if (contentType) {
         if (contentType.includes('json')) {
-          respJson = await tryResp.json();
-          me.responseText = JSON.stringify(respJson, null, 2);
+          if (RegExp('charset=([^"\']+)').test(contentType)) {
```

Comment:
A couple stylistic suggestions:
- Use the inline regex constructor `/.../`, which makes this more concise and avoids the need to escaping the `'`
- Remove the capturing group `(...)`, which isn't used for anything anyway

Both of these changes IMO make the code more readable (shorter, and less of a symbol soup):

```suggestion
          if (/charset=[^"']+/).test(contentType)) {
```